core/control-plane/contract-core/src/main/java/org/eclipse/edc/connector/contract/ContractNegotiationCommandExtension.java

```diff
@@ -30,10 +30,12 @@
  * handlers the core provides.
  */
 @CoreExtension
-@Provides({ CommandHandlerRegistry.class })
-@Extension(value = "Contract Negotiation command handler")
+@Provides({CommandHandlerRegistry.class})
+@Extension(ContractNegotiationCommandExtension.NAME)
 public class ContractNegotiationCommandExtension implements ServiceExtension {
 
+    public static final String NAME = "Contract Negotiation command handler";
```

Comment:
please override the `name` method and make it return the `NAME` constant as well